### PR TITLE
Pull request for Testing Tic-Tac-Toe

### PR DESCRIPTION
**Pull request by RoodzFernando**
# Test implemented
* Dom manipulation
* When the game is a draw
* When one played has won

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        Gameboard.table[i] = player2.marker;
+      }
+    }
+    notif.textContent = `${currentPlayer.name} wins`;
+    expect(Gameboard.table).toEqual([
+      'X', 'O', 'X',
+      'O', 'X', 'O',
+      'X', 'O', 'X'
+    ]);
+  });
```
Comment (maelfosso) [inline src/index.test.js:84]: What are you testing here? 
You affected value in table and check the results. There is none methods tested. 
Re-think your test, please.

File: src/index.test.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      if (currentPlayer === player1) {
+        currentPlayer = player2;
+      } else {
+        currentPlayer = player1;
+      }
+    }
+    changeCurrentPlayer();
+    expect(currentPlayer).toEqual(player2);
+  });
+});
```
Comment (maelfosso) [inline src/index.test.js:109]: Why don't you directly use the `game#changePlayer` ?
Test it directly!!

You imported the `game` from `index.js` but didn't use it. Why?
Instantiate it, make two or three turn and check who the current player is.

File: .eslintrc.json [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-    "no-use-before-define": ["error", {
-        "functions": false,
-        "classes": false,
-        "variables": false
- }]
-  }
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2020": true
```
Comment (Torres-ssf) [inline .eslintrc.json:5]: - [ ] Eslint it's not recognizing Jest variables, we can fix that adding it to the env object here. `"jest": true`

File: .stickler.yml [hunk 4]
```diff
@@ -10,3 +10,5 @@ files:
     ignore:
         - 'src/generated/*'
         - 'webroot/bower_components/*'
+        - 'dist/*'
+        - 'src/*'
```
Comment (Torres-ssf) [inline .stickler.yml:14]: - [ ] src/ directory should not be ignored by the sticker. You can ignore the test file or move it to another directory. There are some linter errors in your index file that need to be fixed.

**Pull request merged**